lib/creator/yeoman/utils/entry.js

```diff
@@ -0,0 +1,50 @@
+const Input = require('webpack-addons').Input;
+
+module.exports = (self, answer) => {
+	return new Promise( (resolve) => {
```

Comment:
You're assigning the promise to a value instead of directly calling it 💯  Thanks! Just tested it and it has the behaviour I wanted. When I did it without referencing an object the callbacks for multiple entries were all outputting to the terminal. Thanks for reviewing! Pushing a commit shortly. Do you mind doing  a quick overview after that?